.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main", "tests-in-ci" ]
```

Comment:
Would remove this branch before committing - this is allowing results to show up in https://github.com/bgrins/Speedometer/actions